cpu/esp8266/vendor/esp-idf/esp8266/source/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_ESP_IDF_ESP8266
+    bool
+    depends on TEST_KCONFIG
+    depends on MODULE_ESP_IDF
```

Comment:
```suggestion
    depends on HAS_ARCH_ESP8266
    depends on MODULE_ESP_IDF
```

Reply:
Added in e38f6154042eae0f189239e4e10a847321edae65